core/client/fs/src/main/java/alluxio/client/file/cache/DefaultMetaStore.java

```diff
@@ -74,5 +84,16 @@ public void reset() {
     mPages.set(0);
     mBytes.set(0);
     mPageMap.clear();
+    mEvictor.reset();
+  }
+
+  @Override
+  @Nullable
+  public PageInfo evict() {
+    PageId victim = mEvictor.evict();
+    if (victim == null) {
+      return null;
+    }
+    return mPageMap.get(victim);
```

Comment:
If this is `null`, do we need to do something to update the evictor state?

Reply:
good catch. In theory, this should never happen if evictor is implemented correctly.
but no one can guarantee that. corresponding error log and cleanup added